src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,107 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static final String VALID_LOGIN = "Username";
+    private static final String ANOTHER_VALID_LOGIN = "Username1";
+    private static final String ONE_MORE_LOGIN = "Username2";
+    private static final String VALID_PASSWORD = "Password";
+    private static final String UNVALID_LOGIN = "12345";
+    private static final String UNVALID_PASSWORD = "12345";
+    private static final int VALID_AGE = 18;
+    private static RegistrationService registrationService;
+    private static Storage storage;
+    private static User validUser;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        storage = new Storage();
+        validUser = new User();
```

Comment:
1: You don`t need to create a new Storage instance it is already imported in your class;
2: registrationService and User can both be initialised as class fields